docs/docsource/operation/Architecture/CloudServer.rst

```diff
@@ -94,28 +94,28 @@ use cases.
 
 -  **Healthcheck**
 
-   Currently, a liveness probe calls /\_/healthcheck/deep. Services that
-   expose readiness can also get a readiness probe.
+   Currently, a liveness probe calls ``/\_/healthcheck/deep``. Services
+   that expose readiness can also get a readiness probe.
 
 -  **Metrics Collection**
 
-   These metrics are valid on all NodeJS-based services:
+   These metrics are valid on all Node.js-based services:
 
 .. tabularcolumns:: X{0.45\textwidth}X{0.45\textwidth}
-.. table:: NodeJS Process General Metrics
+.. table:: Node.js Process General Metrics
 
    +-----------------------------------------------+---------------------------------------------------------+
    | Metric                                        | Description                                             |
    +===============================================+=========================================================+
-   | nodejs\_version\_info                         | NodeJS Version info                                     |
+   | nodejs\_version\_info                         | Node.js Version info                                    |
```

Comment:
`Version` probably doesn't need to be capitalized, but not an important detail.

Reply:
fair spot.